.storybook/themes/news.scss

```diff
@@ -976,6 +982,12 @@
 						),
 					),
 				),
+				links-bar: (
+					gap: var(--global-spacing-2),
+				),
+				links-bar-menu-link: (
+					text-decoration: none,
+				),
```

Comment:
@malavikakoppula : Are these properties at `links-bar` and `links-bar-menu-link` needed at `desktop()` since they are set at `default()` and do not change?

Reply:
Styles are same for both `default() `as well as `desktop()`

<img width="954" alt="image" src="https://user-images.githubusercontent.com/83021791/172910899-8f293f05-b1ae-46bb-a6f5-063513618f53.png">

Reply:
Yes. The CSS is mobile-first, so `desktop()` will use the values in `default()` (default being "mobile").